### PR TITLE
Day 1 - Part 1 Solution

### PR DESCRIPTION
**Pull request by aolszowka**
https://adventofcode.com/2021/day/1

I approached this solution by assuming that I would have all inputs ahead of time (instead of leveraging a Enumerable or Pipeline based approach).

I special case the first value to avoid having additional `if` checks at the cost of utilizing a `for` loop (meaning I needed to populate the entire array ahead of time). If I were to switch this to an Enumerable implementation I would have to restructure this significantly.

I leverage PowerShell's `-Verbose` logging to give a similar output to the example.

The specification was vague on what should occur if the values are the same (my code will ignore them; but in verbose will indicate this scenario).

### CODE REVIEW

**Comment by aolszowka** [comment]
Here is my current output:

```
VERBOSE: 199 (N/A - no previous measurement)
VERBOSE: 200 (increased)
VERBOSE: 208 (increased)
VERBOSE: 210 (increased)
VERBOSE: 200 (decreased)
VERBOSE: 207 (increased)
VERBOSE: 240 (increased)
VERBOSE: 269 (increased)
VERBOSE: 260 (decreased)
VERBOSE: 263 (increased)
7
```

File: Day1.ps1 [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        Write-Verbose -Message "$($Inputs[0]) (N/A - no previous measurement)"
+
+        # Now enumerate though each one
+        for ($i = 1; $i -lt $Inputs.Count; $i++) {
+            if ($Inputs[$i] -gt $Inputs[$i - 1]) {
+                Write-Verbose -Message "$($Inputs[$i]) (increased)"
+                $measurementsLargerThanPrevious++
+            }
+            elseif ($Inputs[$i] -eq $Inputs[$i - 1]) {
+                # The specification didn't say what to do in this scenario? Be silent?
```
Comment (nickkimbrough) [inline Day1.ps1:22]: From what I understand about AoC if the specs don't mention it, it's safe to assume the scenario won't happen.

Comment (aolszowka) [reply Day1.ps1:22]: Sounds legit; I just realized that once you register and submit something you get "more" of the puzzle; I'm trying to figure out how I want to lay this out....

Maybe a Day1.Part1.ps1 vs Day1.ps1; in theory if I didn't squash I could have a creative Git History that shows this; what did you do?

Comment (nickkimbrough) [reply Day1.ps1:22]: I didn't spend too much thought on that part, each puzzle I have has 1 file that has output for part 1 and 2

**Pull request merged**